app/src/main/java/com/vci/vectorcamapp/intake/presentation/IntakeScreen.kt

```diff
@@ -498,8 +519,20 @@ fun IntakeScreen(
 
                     TextEntryField(
                         label = "Number of LLINs Available",
-                        value = if (surveillanceForm.numLlinsAvailable == 0) "" else surveillanceForm.numLlinsAvailable.toString(),
-                        onValueChange = { onAction(IntakeAction.EnterNumLlinsAvailable(it.filter { character -> character.isDigit() })) },
+                        value = (surveillanceForm.numLlinsAvailable ?: 0).toString(),
```

Comment:
Warning: Elvis operator (?:) always returns the left operand of non-nullable type Int